src/constants/userConstants.js

```diff
@@ -0,0 +1,7 @@
+const STANDARD = 'STANDARD';
+const ADMIN = 'ADMIN';
+
+export default {
+  STANDARD,
+  ADMIN,
+};
```

Comment:
These are already defined in auth/constants

Reply:
Ah, thank you @florisdobber, somehow I missed these when looking.

@jackblanc the backend returns something like:
```
{
    "firstName": "Justin",
    "lastName": "Konecny",
    "username": "jkonecny",
    "email": "justin@gmail.com",
    "privilegeLevel": "ADMIN"
}
```

Reply:
@florisdobber thanks--I don't know how I missed that.
@jackblanc yep yep, my mistake, I'm fixing that now!